src/app/@ansyn/menu-items/public_api.ts

```diff
@@ -1,3 +1,6 @@
+import { SetAlgorithmTaskDrawIndicator } from './algorithms/actions/algorithms.actions';
```

Comment:
combine "algorithms.actions"

Reply:
done (the "import" statement seems to be redundant)